mods/persistence/game/machinery/machinery.dm

```diff
@@ -11,4 +11,73 @@
 
 /obj/machinery/before_save()
 	. = ..()
-	initial_access = list() // Remove initial_access so that access isn't wiped on load.
\ No newline at end of file
+	initial_access = list() // Remove initial_access so that access isn't wiped on load.
+
+/obj/machinery/power/tracker/after_deserialize()
+	. = ..()
+	connect_to_network() //There's a powernet mismatch sometimes after loading the tracker
```

Comment:
Does this do anything? Trackers already call connect_to_network in Initialize(), and if the powernets somehow change between this and Initialize(), it's going to be a member of two seperate powernets.

Reply:
I don't think that was meant to be commited actually.. Might have to revert that. 
Solars are way more busted than that could ever fix, and I've been working on something bigger. Its busted on the upstream level at that.